tdrs-backend/tdpservice/parsers/schema_defs/tanf/t1.py

```diff
@@ -1,114 +1,268 @@
+                  Field(item="6", name='CASE_NUMBER', type='string', startIndex=8, endIndex=19,
+                        required=True, validators=[
+                              validators.isAlphaNumeric(),
+                              validators.notMatches('_'*11),
+                              validators.notEmpty(),
+                        ]),
+                  Field(item="2", name='COUNTY_FIPS_CODE', type='string', startIndex=19, endIndex=22,
+                        required=True, validators=[
+                              validators.isNumber(),
+                        ]),
+                  Field(item="5", name='STRATUM', type='string', startIndex=22, endIndex=24,
+                        required=True, validators=[
+                              validators.isInStringRange(0, 99),
+                        ]),
+                  Field(item="7", name='ZIP_CODE', type='string', startIndex=24, endIndex=29,
+                        required=True, validators=[
+                              validators.isNumber(),
+                        ]),
+                  Field(item="8", name='FUNDING_STREAM', type='number', startIndex=29, endIndex=30,
+                        required=True, validators=[
+                              validators.isInLimits(1, 3),
+                        ]),
+                  Field(item="9", name='DISPOSITION', type='number', startIndex=30, endIndex=31,
+                        required=True, validators=[
+                              validators.oneOf([1, 2]),
+                        ]),
+                  Field(item="10", name='NEW_APPLICANT', type='number', startIndex=31, endIndex=32,
+                        required=True, validators=[
+                              validators.oneOf([1, 2]),
+                        ]),
+                  Field(item="11", name='NBR_FAMILY_MEMBERS', type='number', startIndex=32, endIndex=34,
+                        required=True, validators=[
+                              validators.isLargerThan(0),
+                        ]),
+                  Field(item="12", name='FAMILY_TYPE', type='number', startIndex=34, endIndex=35,
+                        required=True, validators=[
+                              validators.isInLimits(1, 3),
+                        ]),
+                  Field(item="13", name='RECEIVES_SUB_HOUSING', type='number', startIndex=35, endIndex=36,
+                        required=True, validators=[
+                              validators.isInLimits(1, 3),
+                        ]),
+                  Field(item="14", name='RECEIVES_MED_ASSISTANCE', type='number', startIndex=36, endIndex=37,
+                        required=True, validators=[
+                              validators.isInLimits(1, 2),
+                        ]),
+                  Field(item="15", name='RECEIVES_FOOD_STAMPS', type='number', startIndex=37, endIndex=38,
+                        required=True, validators=[
+                              validators.isInLimits(1, 2),
```

Comment:
```suggestion
                              validators.isInLimits(0, 2),
```
`0` added since no longer in use.